kotsadm/web/src/components/troubleshoot/SupportBundleList.jsx

```diff
@@ -36,14 +36,19 @@ class SupportBundleList extends React.Component {
     })
       .then(async (res) => {
         const response = await res.json();
-        this.setState({
-          supportBundles: response.supportBundles,
-          loading: false,
-        });
+        if(!res.ok) {
+          this.setState({ loading: false, errorMsg: `Unable to get list of bundles: ${response.error}` });
+        }
+        if(res.ok) {
```

Comment:
same here, how about adding a `return` to the `!res.ok` instead of adding another check here? just to make this cleaner